experiments/ClimaCore/sea-ice/README.md

```diff
@@ -0,0 +1,71 @@
+# **Simple Sea Ice Model**
+
+# Semtner's Zero-layer Model
+
+Sea ice is approximated as a 0-layer slab. This means that the model does not occupy a particular gridpoint between the mixed layer (ocean) and the atmosphere. It is essentially a column-wise ODE, with no horizontal transport between the columns. In the absence of ice, the sea-ice model reduces to the slab ocean formulation. The ice is assumed to have a negligible heat capacity, so there is no heat storage within it. The only store of energy comes from the ice thickness.
+
+We followed the FMS implementation as in [Zhang et al 22](https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2021MS002671), which can be found on [GitHub](https://github.com/sally-xiyue/fms-idealized/blob/sea_ice_v1.0/exp/sea_ice/srcmods/mixed_layer.f90) and is itself a modification of the 0-layer model of [Semtner 1976](https://www.atmosp.physics.utoronto.ca/people/guido/PHY2502/articles/seaice-landice/AJSemtner_1976.pdf) (appendix). Chronologically, the algorithm follow these steps:
+
+## 1. Ice thickness, $h_i$
+$$
+L_i \frac{dh_i}{dt} = F_{atm} - F_{base}
+$$
+- with the latent heat of fusion, $L_i=3 \times 10^8$ J / m3, and where the (upward-pointing) flux into the atmosphere is
+$$
+F_{atm} = F_{rad} + F_{SH} + F_{LH}
+$$
+- $F_{atm}$ is obtained from the atmospheric model (via the coupler). The flux at the ice base into the mixed layer is
+$$
+F_{base} = F_0(T_{ml} - T_{melt})
+$$
+- where $T_{melt} = 273.16$ K is the freezing temperature, which sets the temperature of the entire isothermal ice slab, and the basal heat coefficient $F_0 = 120$ W /m2 / K. 
+
+## 2. Ocean mixed layer temperature, $T_{ml}$
+- $T_{ml}$ is the standard slab ocean formulation in ice-free conditions:
+$$
+\rho_w c_w h_{ml}\frac{dT_{ml}}{dt} = - F_{atm}
+$$
+- while ice-covered conditions require that:
+$$
+\rho_w c_w h_{ml}\frac{dT_{ml}}{dt} = - F_{base}
+$$
+
+## 3. Transition from no ice to ice
+- When the updated $T_{ml}^{t+1} < T_{melt}$, set $T_{ml}^{t+1} = T_{melt}$ and grow ice via $h_i$ due to the corresponding energy deficit.
+- When the updated $h_i^{t+1} <= 0$ from a non-zero $h_i^t$, adjust $h_i^{t+1} = 0$ and use the surplus energy to warm the mixed layer.
+
+## 4. Surface temperature ($T_s$)
```

Comment:
If there is ice, T_s = surface temp of ice? Does that contradict the statement earlier that the ice is an isothermal slab? It seems like we have ice with surf temp = T_s, bottom temp  = T_freeze, and flux at surface =  F_i = k_i \frac{T_{melt} - T_s}{h_i}, flux at bottom = F_base, but we assume F_atm = F_i.

When the surface is ice free, is T_s = T_ml? Should mention that here?